client/widgets/test-execution-time/time.directive.js

```diff
@@ -0,0 +1,124 @@
+angular.module('probedock.testExecutionTimeWidget').directive('testExecutionTimeWidget', function() {
+  return {
+    restrict: 'E',
+    controller: 'TestExecutionTimeWidgetCtrl',
+    templateUrl: '/templates/widgets/test-execution-time/time.template.html',
+    scope: {
+      organization: '=',
+      test: '='
+    }
+  };
+}).controller('TestExecutionTimeWidgetCtrl', function(api, $scope, projectVersions) {
+
+  var page = 1;
+
+  $scope.pageSize = 25;
+  $scope.maxResults = 100;
+
+  $scope.params = {};
+  $scope.results = [];
+  $scope.loading = true;
+
+  $scope.chart = {
+    data: [],
+    labels: [],
+    options: {
+      pointHitDetectionRadius: 5,
+      showTooltips: false,
+      pointDot: false,
+      datasetFill: false,
+      scaleShowVerticalLines: false,
+
+      /*
+       * Fix for space issue in the Y axis labels
+       * see: https://github.com/nnnick/Chart.js/issues/729
+       * see: http://stackoverflow.com/questions/26498171/how-do-i-prevent-the-scale-labels-from-being-cut-off-in-chartjs
+       */
+      scaleLabel: function (object) {
+        return '  ' + object.value;
+      }
+    }
+  };
+
+
+  $scope.$watch('test', function(value) {
+    if (value) {
+      initFetchResults();
+    }
+  });
+
+  $scope.$watch('params', function(value) {
+    if (!_.isEmpty(value)) {
+      initFetchResults();
+    }
+  }, true);
+
+
+  $scope.fetchMore = function() {
```

Comment:
Addendum 2: tested locally. It seems to look good, at least without the dots that other charts have, but I think I would actually like it better if it simply displayed 100 results outright. It seems to me that the most results are displayed, the better chance you have of seeing a trend in the execution time, which is the purpose of this widget. Showing only 25 by default seems like a waste.